coder-sdk/test/login_cmd.go

```diff
@@ -0,0 +1,51 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"net/url"
+
+	"cdr.dev/coder-cli/coder-sdk"
+)
+
+func main() {
```

Comment:
Why is this a main package if it's for testing?

Reply:
oops, did not mean to check this in